src/simplewallet/simplewallet.cpp

```diff
@@ -3621,38 +3426,18 @@ simple_wallet::simple_wallet()
                            boost::bind(&simple_wallet::on_command, this, &simple_wallet::set_ring, _1),
                            tr(USAGE_SET_RING),
                            tr("Set the ring used for a given key image, so it can be reused in a fork"));
-  m_cmd_binder.set_handler("unset_ring",
```

Comment:
Just thinking aloud, make of it what you like: When I code, and delete something that was there for a long time, I do not just delete it, but in the same place of the deleted code I add comments about the deleted code, telling something *what* was once here, and/or *why* it was deleted. Here, just as an example, you could leave info what drove the decision to finally delete these commands in question.